docs/1.34/faq/migrate-to-self-hosted-fq15.mdx

```diff
@@ -26,9 +26,9 @@ Today, we have hundreds of thousands of developers using the [Prisma ORM](https:
 
 First, identify if you're using Prisma 1 Cloud at all!
 
-1. Open `prisma.yml` in your project.
-2. Find the `endpoint` property.
-3. You are using Prisma 1 Cloud if the `endpoint` property includes `prisma.sh`, and should continue reading this tutorial.
+1. Open your project locally.
+2. Inspect `prisma.yml` and any environment variables your build process is using.
+3. If your project uses a `PRISMA_MANAGEMENT_API_SECRET`, then you are **not** using Prisma 1 Cloud. You can verify this by using `prisma1 logout` from the command line, regenerating your Prisma 1 client, and attempting to deploy or access a service.
```

Comment:
Could they not still be using P1 Cloud for management of that server and service?